spec/views/hq/sponsors/index_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
You have to find the Array and replace it with a relation.  So somewhere, instead of `[]` or something like that, it needs to be `Sponsor.none` or similar